libs/lb/lb-rs/src/service/log_service.rs

```diff
@@ -12,28 +12,39 @@ pub fn init(config: &Config) -> LbResult<()> {
         let lockbook_log_level = env::var("LOG_LEVEL")
             .ok()
             .and_then(|s| s.as_str().parse().ok())
-            .unwrap_or(LevelFilter::DEBUG);
+            .unwrap_or(LevelFilter::TRACE);
```

Comment:
I don't think this is a mergable change, it will result in an explosion of log volume that I don't think most users want to incur.

Reply:
You could consider overwriting this at configuration time any number of ways, but as it is, most users aren't going to want this, and it will force us to implement something like log rotation sooner